Supplement the kernel's .gitignore

From: Curt Brune <curt@cumulusnetworks.com>

Copyright (C) 2015 david_yang <david_yang@accton.com>

SPDX-License-Identifier:     GPL-2.0

- Ignore platform device tree blob files
- vmlinux.strip
---
 .gitignore                   |    1 +
 arch/powerpc/boot/.gitignore |    3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index 7e9932e..879800d 100644
--- a/.gitignore
+++ b/.gitignore
@@ -41,6 +41,7 @@ modules.builtin
 /TAGS
 /linux
 /vmlinux
+/vmlinux.strip
 /vmlinuz
 /System.map
 /Module.markers
diff --git a/arch/powerpc/boot/.gitignore b/arch/powerpc/boot/.gitignore
index 554734f..af18d86 100644
--- a/arch/powerpc/boot/.gitignore
+++ b/arch/powerpc/boot/.gitignore
@@ -17,6 +17,7 @@ uImage
 cuImage.*
 dtbImage.*
 treeImage.*
+vmlinux.strip
 zImage
 zImage.initrd
 zImage.bin.*
@@ -40,4 +41,4 @@ fdt_sw.c
 fdt_wip.c
 libfdt.h
 libfdt_internal.h
-
+*.dtb
